pkgs/by-name/yu/yunfaavatar/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  bash,
+  imagemagick,
+  makeWrapper,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "yunfaavatar";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "yunfachi";
+    repo = "yunfaAvatar";
+    rev = version;
+    hash = "sha256-s381fD+Zr4CdeHCz8glvC+0wnDiFzn32/iFSrc5p05U=";
+  };
+
+  strictDeps = true;
+  buildInputs = [bash imagemagick];
```

Comment:
These still aren't necessary, bash is part of stdenv, and you're referring to imagemagick in your `postInstall` directly.

Reply:
so i need to delete lines 20 and 21?

Reply:
You can keep `strictDeps` (though whether it's useful given you only have one dependency is debatable), you don't need the `buildInputs`.

Reply:
I removed `buildInputs` and did `postInstall`, can you please review?